Tests/instance_notifier.py

```diff
@@ -27,6 +27,7 @@ def options_handler():
 def install_new_content(client, server):
     prints_manager = ParallelPrintsManager(0)
```

Comment:
missed that the first time
```suggestion
    prints_manager = ParallelPrintsManager(1)
```